doc/content/help/faq.md

```diff
@@ -1,14 +1,136 @@
+
+```
+#!/bin/bash
+#PBS -M FIRSTNAME.LASTNAME@inl.gov
+#PBS -m abe
+#PBS -N Example03_hpc
+#PBS -P moose
+#PBS -l select=4:ncpus=36:mpiprocs=36
+#PBS -l walltime=00:90:00
+
+JOB_NUM=${PBS_JOBID%%\.*}
+
+cd $PBS_O_WORKDIR
+
+\rm -f out
+date > out
+MV2_ENABLE_AFFINITY=0 mpiexec ~/projects/mastodon/mastodon-opt -i shear_beam_Isoil_free_field.i >> out
+date >> out
+```
+
+- You can re-use the above file with the following changes:    
+
+  - `FIRSTNAME.LASTNAME@inl.gov` is your INL email ID.
+  - `Example03_hpc` is the name of the job.
+  - `select=4:ncpus=36:mpiprocs=36` means you requested 4 nodes, 36 cpus, and 36 processors. You can increase or decrease these numbers depending on the hpc machine.
+  - `walltime=00:90:00` is the wall time above which the jobs gets terminated. The format is Days:Hours:Minutes. Larger the wall time, more the delay in getting the job started, in general.
+  - `MV2_ENABLE_AFFINITY=0 mpiexec ~/projects/mastodon/mastodon-opt -i shear_beam_Isoil_free_field.i >> out` has the name of the input file. Use the input file name you are interested in.
+
+- Finally, save this file `Example03_hpc.sh` in the folder in which the input file is located. Then, run the following.
+
+```
+qsub Example03_hpc.sh
+```
+
+#### 4. Some or all my tests are failing, what should I do?
+
+There might be one of several reasons for this:
+
+- You did not compile MASTODON. Compile MASTODON using `make` command.
+
+
+- If the errors while running tests are related to libmesh, your libmesh is out of date. Update and rebuild libmesh using the following command in the MASTDON folder. Then compile MASTODON and try running the tests.
+
+```
+./moose/scripts/update_and_rebuild_libmesh.sh
+```
+
+- You might have accidentally made some changes to MASTODON source code. To undo these changes use the below command. Then update and rebuild libmesh, compile MASTODON, and run the tests as described in [Getting Started/MacOS (Mojave)](https://mooseframework.inl.gov/mastodon/getting_started/macos_mojave.html).  
+
+```
+git clean -dxf
+```
+
+#### 5. How to write an input file from scratch?
+
+Generally, we do not recommend doing this. On the [Examples](https://www.mooseframework.org/mastodon/examples/index.html) page, a variety of problems related to seismic soil structure interaction of nuclear structures are discussed. We recommend downloading the input file from one of those problems and modifying it to suit your own problem. In this process, the MASTODON [User Manual](https://www.mooseframework.org/mastodon/manuals/user/index.html) and [Syntax](https://www.mooseframework.org/mastodon/syntax/index.html) pages might also help.
+
```

Comment:
Also there are a number of input files checked into the tests directories within MOOSE and MASTODON